website/docs/integrations/vscode.mdx

```diff
@@ -22,72 +23,125 @@ description: ConfigCat Visual Studio Code extension. This is a step-by-step guid
 - Connect a Config to your Workspace.
 - Open a Config on ConfigCat Dashboard.
 
+<img src="/docs/assets/vscode/ff_setting_view.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags Visual Studio Code Extension" />
+
 ## Install extension
 
-### Visual Studio Code Marketplace
+### Install from Visual Studio Code Marketplace
 
 1. Open [ConfigCat Feature Flags Extension](https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags) at Visual Studio Marketplace.
```

Comment:
Ha nem találod túl erőltetettnek, akkr ezt javaslom:

In the Visual Studio Marketplace, open the [ConfigCat Feature Flags Extension](https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags).